modules/programs/nushell.nix

```
@@ -55,6 +49,24 @@ in
 
     package = lib.mkPackageOption pkgs "nushell" { nullable = true; };
 
+    configDir = lib.mkOption {
+      type = types.path;
```

```suggestion
      type = types.either types.str types.path;
```

Needs to support a basic string, too. 

>error: A definition for option `programs.nushell.configDir' is not of type `absolute path'. Definition values:
       - In `/nix/store/crj3fd4mm48jbl2bb4ml37apsib68ij4-source/tests/modules/programs/nushell/config-dir.nix': "Library/Application Support/nushell"

---

Rebased to `master` and applied that change in my latest commit. Thanks for reporting the error, I don't have a darwin machine :)

This made me wonder if allowing any string is better than setting the default as an absolute path tho:

```nix
      default =
        if pkgs.stdenv.isDarwin && !config.xdg.enable then
          "${config.home.homeDirectory}/Library/Application Support/nushell"
        else
          "${config.xdg.configHome}/nushell";
```